README.md

```diff
@@ -116,61 +154,77 @@ for _, lsp in ipairs(servers) do
+  Defaults to all servers matching current buffer filetype.
+- `:LspStop <client_id>` Defaults to stopping all buffer clients.
+- `:LspRestart <client_id>` Defaults to restarting all buffer clients.
 
 ## The wiki
+
 Please see the [wiki](https://github.com/neovim/nvim-lspconfig/wiki) for additional topics, including:
 
-* [Installing language servers automatically](https://github.com/neovim/nvim-lspconfig/wiki/Installing-language-servers-automatically)
-* [Snippets support](https://github.com/neovim/nvim-lspconfig/wiki/Snippets-support)
-* [Project local settings](https://github.com/neovim/nvim-lspconfig/wiki/Project-local-settings)
-* [Recommended plugins for enhanced language server features](https://github.com/neovim/nvim-lspconfig/wiki/Language-specific-plugins)
+- [Installing language servers automatically](https://github.com/neovim/nvim-lspconfig/wiki/Installing-language-servers-automatically)
+- [Snippets support](https://github.com/neovim/nvim-lspconfig/wiki/Snippets-support)
+- [Project local settings](https://github.com/neovim/nvim-lspconfig/wiki/Project-local-settings)
+- [Recommended plugins for enhanced language server features](https://github.com/neovim/nvim-lspconfig/wiki/Language-specific-plugins)
 
 ## Windows
 
-In order for neovim to launch certain executables on Windows, it must append `.cmd` to the command name. To work around this, manually append `.cmd` to the entry `cmd` in a given plugin's setup{} call.  
+In order for Neovim to launch certain executables on Windows, it must append `.cmd` to the command name.
+To work around this, manually append `.cmd` to the entry `cmd` in a given plugin's `setup{}` call.
 
 ## Contributions
-If you are missing a language server on the list in [CONFIG.md](CONFIG.md), contributing
-a new configuration for it would be appreciated. You can follow these steps:
+
+If you are missing a language server on the list in [CONFIG.md](CONFIG.md), contributing a new configuration for it would be appreciated.
+You can follow these steps:
 
 1. Read [CONTRIBUTING.md](CONTRIBUTING.md).
-2. Choose a language from [the coc.nvim wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers) or
-  [emacs-lsp](https://github.com/emacs-lsp/lsp-mode#supported-languages).
-3. Create a new file at `lua/lspconfig/SERVER_NAME.lua`.
-   - Copy an [existing config](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/)
-     to get started. Most configs are simple. For an extensive example see
-     [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
-4. Ask questions on our [Discourse](https://neovim.discourse.group/c/7-category/7) or in the [Neovim Gitter](https://gitter.im/neovim/neovim).
+1. Choose a language from [the coc.nvim wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers) or [emacs-lsp](https://github.com/emacs-lsp/lsp-mode#supported-languages).
```

Comment:
Is there a reason you changed this to repeat 1.?

Reply:
Just standard markdownlint fix: they will auto-update so there's no need to explicitly say step numbers.

Reply:
Isn't this confusing when people open markdown with a text editor like vim/neovim that doesn't transparently render the numbers?

Reply:
Could be I guess, I wouldn't do it that way but it's your repo.